jsx/StaticDataTable.js

```diff
@@ -303,6 +303,54 @@ class StaticDataTable extends Component {
   getSortedRows() {
     const index = [];
 
+    // TODO: data column type check should probably be done once at init,
+    // to find which columns have mixed types.
+    // First pass to check if there is a mix of types in the values
+    const typesFound = [];
+    // is the current sorted column with mixed type?
+    let isMixedType = false;
+    // not the default column
+    if (this.state.SortColumn !== -1) {
+      // only push string or number types, rest is considered undefined
+      //
+      // break out of this loop once we encounter two mixed types:
+      // number and string inside the sorting column
+      for (let i = 0; i < this.props.Data.length; i += 1) {
+        // value
+        let val = this.props.Data[i][this.state.SortColumn] || undefined;
```

Comment:
That would cause problems if the data is `0`, I think you should use the nullish-coalescing operator `??` instead.

Reply:
Maybe I am missing the point here, or there is something I did not catch. 
Does `data is 0` means the data array is empty?
If data is empty, the loop will not execute?

Reply:
Oh, forgive my imprecise formulation. By data I meant the whole expression `this.props.Data[i][this.state.SortColumn]`. So if the cell contains `0` this line would transform it to `undefined`. But after giving it more thought, I still don't get the point of this line (not blocking on it though), in which case is this line useful ?